docs/api/breadcrumbs.md

```diff
@@ -51,6 +51,40 @@ import CustomSeparators from '@site/static/usage/breadcrumbs/icons/custom-separa
 
 <CustomSeparators />
 
+## Collapsing Items
+
+### Max Items
+
+If there are more items than the value of `max-items`, the breadcrumbs will be collapsed. By default, only the first and last items will be shown.
```

Comment:
```suggestion
If there are more items than the value of `maxItems`, the breadcrumbs will be collapsed. By default, only the first and last items will be shown.
```

We typically refer to the property names using camel case as that is how they are documented in the "Properties" section.